docs/getting-started-on-the-raspberry-pi.md

```diff
@@ -0,0 +1,104 @@
+# Getting Started on the Raspberry Pi 3
+
+This guide assumes you are using the [Raspbian image]. It should work on other
+distributions as well, but some details may be different.
+
+We will compile the driver on the Raspberry Pi itself, of course you can also
+cross compile from your system. 
+
+## Download Driver
+
+Get the latest release from
+https://github.com/Sensirion/embedded-uart-sps/releases (sps30-uart-x.y.z.zip)
+
+```bash
+# on the Raspberry Pi
+$ wget https://github.com/Sensirion/embedded-uart-sps/releases/download/1.0.0/sps30-uart-1.0.0.zip
+$ unzip sps30-uart-1.0.0.zip
+```
+
+## Get Port of the Connected Sensor
+
+We need to now the port of the connected sensor. After connecting the sensor
+via USB run the following:
+```bash
+# on the Raspberry Pi
+$ dmesg | grep usb
+```
+This should show a line like this:
+```
+[   63.545635] usb 1-1.3: FTDI USB Serial Device converter now attached to ttyUSB0
+```
+This means the serial device will be reachable as `/dev/ttyUSB0`.
+
+## Adapt the Source Code
+
+Copy the file `sample-implementations/linux/sensirion_uart_implementation.c` to
+the base folder:
+
+```bash
+# on the Raspberry Pi
+$ cd sps30-uart-1.0.0
+$ cp sample-implementations/linux/sensirion_uart_implementation.c sensirion_uart_implementation.c
+```
+
+Then in `sensirion_uart_implementation.c` change the line with `#define TTYDEV`
+to match the device you got from the step before.
+```c
+#define TTYDEV "/dev/ttyUSB0"
+```
+
+## Compile and Run
+
+Now we are ready to compile and run the sample:
```

Comment:
Add a note that every time the device is unplugged, the devicepath might change and the binary must be recompiled.

Reply:
Didn't happen during my testing, but true that *may* happen.

Reply:
I added a trouble shooting section which explains this among other errors.